requirements/default.in

```diff
@@ -45,7 +45,7 @@ google-api-python-client~=1.8.4
 gunicorn~=20.0.4
 html5lib~=1.0.1
 httplib2~=0.18.1
-https://github.com/glogiotatidis/django-pipeline/archive/103bb549cd5f93b5648e10f1be6c2c6c8c5e9610.tar.gz#egg=django-pipeline
```

Comment:
fyi @glogiotatidis. This commit will reach production in the next couple of weeks so sumo won't be needing this.